app/src/main/java/org/mozilla/focus/ui/theme/FocusTheme.kt

```diff
@@ -48,6 +48,7 @@ val focusColors: FocusColors
 
 private fun darkColorPalette(): FocusColors = FocusColors(
     material = darkColorsMaterial(),
+    accent = PhotonColors.Pink40,
```

Comment:
nit: I think we should choose an explicit name and move away from the old XML naming that was a bit confusing. The name should explicitly reflect it is being used for active controls.